### PR TITLE
Add secret setup step to WhisperX tutorial

### PR DESCRIPTION
**Pull request by odysseus0**

### CODE REVIEW

File: advanced/whisperx/README.md [hunk 1]
```diff
@@ -90,7 +90,32 @@ and invoke transcription or translation as follows:
...
+```txt
+               Secrets
+┏━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┓
+┃ ID                     ┃ Value    ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━┩
+│ HUGGING_FACE_HUB_TOKEN │ (hidden) │
+└────────────────────────┴──────────┘
+```
+
+Now you can proceed to photo creation and deployment by running the following command:
```
Comment (Yangqing) [inline advanced/whisperx/README.md:118]: Would you mind also adding on line 125 the following argument to actually use the secret?
```
lep photon run -n whisperx --resource-shape gpu.a10 --secret HUGGING_FACE_HUB_TOKEN
```

**Review by Yangqing — COMMENTED** [review]
Thanks so much!

File: advanced/whisperx/README.md (hunk 1 above)
Comment (odysseus0) [reply advanced/whisperx/README.md:131]: done! Please see the new commit.

**Comment by Yangqing** [comment]
Lovely, thanks again! Merging.

**Pull request merged**